apps/andi/lib/andi_web/live/dataset_live_view.ex

```diff
@@ -159,12 +180,21 @@ defmodule AndiWeb.DatasetLiveView do
   end
 
   defp filter_remotes(datasets, true), do: datasets
+  defp filter_submitted(datasets, true) do
+    Enum.filter(datasets, fn dataset -> dataset.submission_status == :submitted end)
+  end
+  defp filter_submitted(datasets, false), do: datasets
+
+  defp convert_to_view_models(datasets) do
+    Enum.map(datasets, &to_view_model/1)
+  end
 
   defp to_view_model(dataset) do
     %{
       "id" => dataset.id,
       "org_title" => dataset.business.orgTitle,
       "data_title" => dataset.business.dataTitle,
+      "remote" => dataset.technical.sourceType == "remote",
```

Comment:
oops this can go away. Alternatively, we have a path forward as well for holding `unfiltered_view_models` instead of `datasets`